### PR TITLE
Remove message parser dead code 

### PR DESCRIPTION
**Pull request by kubic71**
Bot responds to two commands:
- network status
- decryption request using context menu

The functionality where the bot parses message content and responds based on it is no longer used and it's a dead code, so it was removed.

### CODE REVIEW

File: revealer_bot/built_in_messages.py [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-    ("multiple", "Yes, everyone is welcome to contribute as many times as they like."),
-    ("number", "NFT numbers will be allocated in order of the largest contribution to the smalled."),
-    ("minimum",
-     "There are two ways to think about contributions. Any contribution is welcome -- but ... have to be more than .1 ETH"),
-    ("usd", "USD is welcomed as a contribution."),
-    ("after",
-     "Yes. Contributions to the artist are welomed after the album has been released. These contributions will not yeald an NFT and they will not be recorded on the album.release block chain."),
-    ("3 ETH", "Once 3 ETH have been contributed, 3 songs will be release."),
-    ("released", "The album is released and open to all once 10 ETH have been contributed."),
-]
```
Comment (KPrasch) [inline revealer_bot/built_in_messages.py:22]: Perhaps some if this information can be used to populate an FAQ on the website/dapp.

**Review by KPrasch — APPROVED** [review]
Indeed, this functionality is more akin to an "FAQ" bot.  Perhaps it can be reintroduced in a more sophisticated way that uses language models someday.

File: revealer_bot/built_in_messages.py (hunk 1 above)
Comment (kubic71) [reply revealer_bot/built_in_messages.py:22]: Okay, I'll create a new issue in the dApp repo.

**Comment by kubic71** [comment]
@KPrasch Fixed conflict, ready for re-review.

**Pull request merged**